src/Components/Loaders/Ellipsis.jsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import styles from "./Ellipsis.module.css";
+
+const Ellipsis = () => {
+  return (
+    <div className={styles.ldioff}>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
+      <div></div>
```

Comment:
어떤 역할을 하는 엘리먼트인지 className을 주면 코드를 이해하기 쉬워져요.
```suggestion
      <div className="dot"></div>
      <div className="dot"></div>
      <div className="dot"></div>
      <div className="dot"></div>
      <div className="dot"></div>
```

Reply:
좋은 지적 감사합니다.